pkg/gardenadm/cmd/token/list/options.go

```diff
@@ -24,4 +26,6 @@ func (o *Options) Validate() error { return nil }
 // Complete completes the options.
 func (o *Options) Complete() error { return nil }
 
-func (o *Options) addFlags(_ *pflag.FlagSet) {}
+func (o *Options) addFlags(fs *pflag.FlagSet) {
+	fs.BoolVarP(&o.WithTokenSecret, "with-token-secret", "", false, "Display the token secret")
```

Comment:
Why do you use the short-cut variant when not specifying a short-cut?

```suggestion
	fs.BoolVar(&o.WithTokenSecret, "with-token-secret", false, "Display the token secret")
```